### PR TITLE
[FIF-164] Catch when tokens expire

### PR DESCRIPTION
**Pull request by jleiva-gap**
1. Login to the buzz site with a registered user.
2. In the dev tools console
image

- Go to Application / SessionStorage.
- Modify the validatingToken key
- Set an invalid key or an expired key.
3. If the key is expired or invalid, redirect to the login screen

### CODE REVIEW

**Review by kurtjohnson-HS — CHANGES_REQUESTED** [review]
I am still getting an error when I navigate to another page. Here is what I did.

1. Log in as projectedfibuzzdemo@gmail.com
2. Navigate to a student's detail page.
3. Alter the validating token - switch the case of the last character.
4. Click the navigation link back to roster.
5. I expected to be redirected back to login, but instead it threw this exception.

image

**Comment by jleiva-gap** [comment]
> 
> 
> I am still getting an error when I navigate to another page. Here is what I did.
> 
>     1. Log in as [projectedfibuzzdemo@gmail.com](mailto:projectedfibuzzdemo@gmail.com)
> 
>     2. Navigate to a student's detail page.
> 
>     3. Alter the validating token - switch the case of the last character.
> 
>     4. Click the navigation link back to roster.
> 
>     5. I expected to be redirected back to login, but instead it threw this exception.
> 
> 
> image

I updated the JWT validation. You must include **_REACT_APP_URI_DISCOVERY=https://accounts.google.com/.well-known/openid-configuration_** in your UI .env file.

**Comment by kurtjohnson-HS** [comment]
> REACT_APP_URI_DISCOVERY=https://accounts.google.com/.well-known/openid-configuration

I will update FIF-299 to make sure we use this URL to create the .env file in UI.

File: EdFi.Buzz.Installer/eng/Windows/example.configuration.json
Comment (kurtjohnson-HS) [inline EdFi.Buzz.Installer/eng/Windows/example.configuration.json:15]: Can you change the tabs to spaces? That might be on your machine.

File: EdFi.Buzz.Installer/eng/Windows/example.configuration.json
Comment (kurtjohnson-HS) [inline EdFi.Buzz.Installer/eng/Windows/example.configuration.json:57]: Interesting that this was changed to add spaces to the end. Your eslint might need a check?

**Review by kurtjohnson-HS — CHANGES_REQUESTED** [review]
Functionality works for me, now that I have added the proper env variable. I just have some spaces recommendations before merging. I approve once those two changes are in.

File: EdFi.Buzz.Installer/eng/Windows/example.configuration.json
Comment (jleiva-gap) [reply EdFi.Buzz.Installer/eng/Windows/example.configuration.json]: Spaces were removed.

**Review by kurtjohnson-HS — APPROVED** [review]

**Pull request merged**
